src/test/java/AudreyTest.java

```diff
@@ -34,10 +34,13 @@ public void setupAudrey() {
             .option("audrey", "true")
             .option("audrey.Project", "tests")
             .option("audrey.Storage", "in_memory")
+            .allowAllAccess(true)
```

Comment:
Is this necessary in the end? If not it's safer to not have it so that "rm -rf /" cannot accidentally end up in tests.

Reply:
R needs access to the native interface and wants to access some files in `/var/`. I made this [a little more restrictive now](https://github.com/rathrio/audrey/pull/6/commits/99eb5fef05d944210d0928f2d33faa6dbdd8f8f0), but I assume ["unrestricted IO operations"](https://github.com/oracle/graal/blob/5ff01c027cf59e4af379b775d38c42ce720ad202/sdk/src/org.graalvm.polyglot/src/org/graalvm/polyglot/Context.java#L936-L938) would allow `rm -rm /`?

Is there a more granular way to restrict file system access?